source/content/guides/drupal-9/drupal-9-hosted-btworkflow/08-database.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Migrate a Site Created With the Pantheon Dashboard to Drupal 9 + Build Tools
+subtitle: Add Database
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-btworkflow/database
+anchorid: database
+editpath: drupal-9/drupal-9-hosted-btworkflow/08-database.md
+---
+
+<Partial file="migrate-add-database.md" />
```

Comment:
Whoops! I missed this in the first version of this doc. The idea here is to export a database from your current Pantheon site and then import it in the new. So, steps 1 and 2 in this partial (that may be used somewhere else) is just going to Pantheon dashboard and exporting a database from the live env.